wave_kinematics/tests/rotation_test.cpp

```diff
@@ -0,0 +1,185 @@
+#include <iostream>
+#include <random>
+
+#include <gtest/gtest.h>
+
+#include "wave/kinematics/numerical_test_functors.hpp"
+#include "wave/kinematics/rotation.hpp"
+#include "wave/utils/math.hpp"
+#include "wave/utils/utils.hpp"
+
+
+class RotationTestFixture : public ::testing::Test {
+public:
+  // Data members used for testing.
+  wave::Rotation rotation1;
+  wave::Rotation rotation1_truth;
+  wave::Rotation rotation2;
+  wave::Rotation rotation3;
+  wave::Rotation rotation1_inverse;
+  wave::Rotation eye3;
+
+  // A vector v1.
+  wave::Vec3 v1;
+  // v1 rotated by rotation1.
+  wave::Vec3 v1_rotated;
+  // True angle axis params for rotation1.
+  wave::Vec3 rotation1_angleaxis_params;
+
+  void SetUp() {
+    wave::Rotation::fromEulerXYZ(0.1, 0.2, 0.3, rotation1);
```

Comment:
ahh ok, so this answers my previous question.  So then the same variables are to be duplicated in the individual tests?